Resources/Meta Analysis/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -1979,9 +1984,9 @@ Form
 
 			IntegerField
 			{
-				label:  		qsTr("Bridge sampling iterations:")
-				name:     		"advanced_bridge_iter"
-				defaultValue: 	10000
+				label:			qsTr("Bridge sampling iterations:")
```

Comment:
this is the maximum number of bridge sampling iterations right? the algorithm terminates early if the r-value converges.

Reply:
yes, that's true, the label is getting a bit too long already and then it starts compressing the second column, not really sure how to address that